bootstrap/Workspace_Management/Generate_Workspace.groovy

```diff
@@ -20,45 +20,77 @@ generateWorkspaceJob.with{
         preBuildCleanup()
         injectPasswords()
         maskPasswords()
-        environmentVariables {
-            env('DC',"${LDAP_ROOTDN}")
-            env('OU_GROUPS','ou=groups')
-            env('OU_PEOPLE','ou=people')
-            env('OUTPUT_FILE','output.ldif')
-        }
-        credentialsBinding {
-            usernamePassword("LDAP_ADMIN_USER", "LDAP_ADMIN_PASSWORD", "adop-ldap-admin")
+        if("${ADOP_LDAP_ENABLED}" == "true")
```

Comment:
This variable ```ADOP_LDAP_ENABLED``` needs to appear:
a) In the compose file for jenkins
b) Injected as a global env variable here: https://github.com/Accenture/adop-jenkins/blob/master/resources/init.groovy.d/adop_general.groovy